config/default/core.entity_form_display.node.hs_private_page.default.yml

```diff
@@ -6,6 +6,7 @@ dependencies:
     - field.field.node.hs_private_page.field_hs_priv_page_components
     - field.field.node.hs_private_page.field_menulink
     - field.field.node.hs_private_page.field_priv_wysiwyg_files
+    - field.field.node.hs_private_page.layout_builder__layout
```

Comment:
It looks like this is also running into the same issues as another PR ([see here](https://github.com/SU-HSDO/suhumsci/pull/1610#pullrequestreview-2281061814)) where this field is being added a dependencies for the display form, but the field configuration doesn't exist. This is what's failing the `test_new_site` test.

Reply:
Thanks, I removed the dependency and related config.